pkg/controller/migcluster/validation.go

```diff
@@ -216,6 +226,58 @@ func (r ReconcileMigCluster) testConnection(cluster *migapi.MigCluster) error {
 	return nil
 }
 
+// Validate the Exposed registry route
+func (r ReconcileMigCluster) validateRegistryRoute(cluster *migapi.MigCluster) error {
+
+	if cluster.Status.HasCriticalCondition() {
+		return nil
+	}
+
+	if cluster.Spec.ExposedRegistryPath != "" {
+		url := "https://" + cluster.Spec.ExposedRegistryPath + "/v2/"
```

Comment:
nit: if we are changing the naming in UI, shall we change this to `ExpectedRegistryHost` as well?

Reply:
Host might be confusing too, though. But ExpectedRegistryHostAndOptionalPort is probably too long.